app/src/lib/types/LocalStorageKeys.ts

```diff
@@ -1,6 +1,13 @@
+import appConfig from "../../config/appConfig";
+
+const getLocalStorageKey = (key: string) => {
+  const ns = appConfig.appNamespace();
+  return ns ? `${ns}-${key}` : key;
```

Comment:
The ns and the key are likely to contain some themselves, and then we might end up with something like `priority-pathogen-authConfig`. It wouldn’t call any problems per se, but best to avoid I think.

A dot or colon or slash would be better as a separator I think.

Reply:
We should be using camelCase for the keys, but agree that dot is probably better as ns might include dashes, if they're going to match the url part.

Reply:
good point, have gone with . instead